Sources/web3swift/Utils/Hooks/NonceMiddleware.swift

```diff
@@ -49,24 +49,25 @@ extension Web3.Utils {
         }
 
         // FIXME: Rewrite this to CodableTransaction
-        func preAssemblyFunction(tx: inout CodableTransaction, contract: EthereumContract) -> (CodableTransaction, EthereumContract, Bool) {
-            guard let from = tx.from else {
-                // do nothing
-                return (tx, contract, true)
-            }
-            guard let knownNonce = self.nonceLookups[from] else {
-                return (tx, contract, true)
-            }
-
-            let newNonce = knownNonce + 1
-
-            self.queue.async {
-                self.nonceLookups[from] = newNonce
-            }
-
-            tx.noncePolicy = .exact(newNonce)
-            return (tx, contract, true)
-        }
+//        func preAssemblyFunction(tx: inout CodableTransaction, contract: EthereumContract) -> (CodableTransaction, EthereumContract, Bool) {
+//            guard let from = tx.from else {
+//                // do nothing
+//                return (tx, contract, true)
+//            }
+//            guard let knownNonce = self.nonceLookups[from] else {
+//                return (tx, contract, true)
+//            }
+//
+//            let newNonce = knownNonce + 1
+//
+//            self.queue.async {
+//                self.nonceLookups[from] = newNonce
+//            }
+//
+//            // FIXME:
+//            tx.noncePolicy = .exact(newNonce)
+//            return (tx, contract, true)
+//        }
```

Comment:
I've remember this one, it the cause of a bug with data encoding. But I thought that @JeneaVranceanu fixed it, so why does it commented out?

Reply:
Not sure but whole `NonceMiddleware` class doesn't seem to be used in the project anyway?

Reply:
I guess it's not, so this one was not issue, just a curiosity, and I think I've been messed up while writing this one.